src/commands/addBinding/settingSteps/LocalAppSettingListStep.ts

```diff
@@ -24,7 +25,6 @@ import { CosmosDBListStep } from './cosmosDB/CosmosDBListStep';
 import { EventHubAuthRuleListStep } from './eventHub/EventHubAuthRuleListStep';
 import { EventHubConnectionCreateStep } from './eventHub/EventHubConnectionCreateStep';
 import { EventHubListStep } from './eventHub/EventHubListStep';
-import { EventHubNamespaceListStep } from './eventHub/EventHubNamespaceListStep';
```

Comment:
Are you missing an import for:  `EventHubsNamespaceListStep`?

Reply:
No idea why that got removed...